semantic-python/semantic-python.cabal

```diff
@@ -20,7 +20,7 @@ tested-with:         GHC == 8.6.5
 
 common haskell
   default-language:    Haskell2010
-  build-depends:       base ^>=4.12
+  build-depends:       base >= 4.13 && < 4.14
```

Comment:
Ditto this—maybe we should be using `>= 4.13 && < 5` like we do in other packages?

Reply:
I think `^>= 4.13` is probably best. We want to dictate what version of GHC is best suited, and the best way to do that is a tight bound on `base`.